frontend/src/main.jsx

```diff
@@ -1,14 +1,27 @@
 import { StrictMode } from 'react'
 import { createRoot } from 'react-dom/client'
-import App  from './App.jsx'
 import { GoogleOAuthProvider } from '@react-oauth/google'
+import { createBrowserRouter, RouterProvider } from 'react-router-dom'
+import ProfilePage from './components/Profile/ProfilePage';
+import App from './App.jsx'
 
 const CLIENT_ID = import.meta.env.VITE_GOOGLE_CLIENT_ID
 
+const router = createBrowserRouter([
+	{
+		path: "/",
+		element: <App />
+	},
+	{
+		path: "/profile",
+		element: <ProfilePage />
```

Comment:
`Profile` should be a private route, you should only be able to access it when the user is logging in.